emails.txt

```diff
@@ -718,23 +997,29 @@ wegwerfmail.org
 wetrainbayarea.com
 wetrainbayarea.org
 wh4f.org
+whatiaas.com
+whatsaas.com
 whyspam.me
 willhackforfood.biz
 willselfdestruct.com
 winemaven.info
+worker.com
+workmail.com
+writeme.com
 wronghead.com
 wuzup.net
 wuzupmail.net
-www.e4ward.com
-www.gishpuppy.com
-www.mailinator.com
 wwwnew.eu
 x.ip6.li
 xagloo.com
 xemaps.com
 xents.com
 xmaily.com
 xoxy.net
+xyzfree.net
+yahoo.com.ph
```

Comment:
I see a bunch that should not have been merged  - ill just revert this PR because its garbage